src/ui/page/channel/view.jsx

```diff
@@ -1,36 +1,87 @@
 // @flow
-import React from 'react';
+import React, { useState } from 'react';
 import { parseURI } from 'lbry-redux';
 import Page from 'component/page';
 import SubscribeButton from 'component/subscribeButton';
 import ShareButton from 'component/shareButton';
 import { Tabs, TabList, Tab, TabPanels, TabPanel } from 'component/common/tabs';
 import { withRouter } from 'react-router';
+import { Form, FormField } from 'component/common/form';
+import Button from 'component/button';
 import { formatLbryUriForWeb } from 'util/uri';
 import ChannelContent from 'component/channelContent';
 import ChannelAbout from 'component/channelAbout';
 import ChannelThumbnail from 'component/channelThumbnail';
+import SelectAsset from '../../component/selectAsset/view';
+import * as ICONS from 'constants/icons';
 
 const PAGE_VIEW_QUERY = `view`;
 const ABOUT_PAGE = `about`;
 
 type Props = {
   uri: string,
   title: ?string,
+  amount: number,
   cover: ?string,
   thumbnail: ?string,
   page: number,
   location: { search: string },
   history: { push: string => void },
   match: { params: { attribute: ?string } },
+  channelIsMine: boolean,
+  description: string,
+  website: string,
+  email: string,
+  balance: number,
+  updateChannel: any => void,
+  tags: Array<string>,
+  locations: Array<string>,
+  languages: Array<string>,
 };
 
 function ChannelPage(props: Props) {
```

Comment:
For now lets just create a `channelEdit` component and put all of the edit stuff in there.